script.js

```diff
@@ -33,4 +34,144 @@ if (bobBMI < 18.5 || bobBMI > 25) {
     console.log("Bob has underweight or overweight");
 } else {
     console.log("Bob has normal weight");
-}
\ No newline at end of file
+}
+
+
+
+//Homework 15
+//Упражнения с тернарным оператором в JavaScript:
+//1 
+const number = 22;
+const numberType = number % 2 !== 0 ? "Odd" : "Even";
+console.log(`The ${number} is ${numberType}`);
+
+
+//2
+const isLoggedIn = true;
+const messageLog = isLoggedIn ? "Logged In" : "Logged Out";
+console.log(messageLog);
+
+
+//3
+const price = 15;
+const quantity = 5;
+const isAvailable = true;
+
+const totalPrice = isAvailable ? price * quantity : "The product is out of stock!"
+console.log(totalPrice);
+ 
+
+//4
+const currentYear = 2023;
+const isLeapYear = (currentYear % 4 === 0 && currentYear % 100 !== 0) || (currentYear % 400 === 0);
+const messageYear = isLeapYear ? "Leap year" : "Not a leap year";
+console.log(messageYear);
+
+
+//Упражнения со switch:
+//1
+let countryCode = "DE";
+switch (countryCode) {
+    case "MD":
+        countryName = "Moldova";
+        break;
+    case "US":
+        countryName = "United States";
+        break;
+    case "FR":
+        countryName = "France";
+        break;
+    case "DE":
+        countryName = "Germany";
+        break;    
+    case "JP":
+        countryName = "Japan";
+        break;
+    case "IN":
+        countryName = "India";
+        break;
+    case "BR":
+        countryName = "Brazil";
+        break;
+    default:
+        countryName = "Unknown country";
+}
+console.log(`The ${countryCode} corresponds to the ${countryName}`);
+
+
+//2
+const score = 96;
+switch (true) {
+    case score >= 97 && score <= 100:
+        grade = "A+";
+        break;
+    case score >= 93 && score <= 96:
+        grade = "A";
+        break;
+    case score >= 90 && score <= 92:
+        grade = "A-";
+        break;
+    case score >= 87 && score <= 89:
+        grade = "B+";
+        break;
+    case score >= 83 && score <= 86:
```

Comment:
No need to check the score to be  <= 86 and the same for all cases. 
If the score wasn't <=86 you wouldn't get into this case and the code would stop on "B+" case

Reply:
It is not an error, but redundant validation

Reply:
> No need to check the score to be <= 86 and the same for all cases. If the score wasn't <=86 you wouldn't get into this case and the code would stop on "B+" case

thanks, now it looks more simple ))